README.md

```diff
@@ -1,53 +1,93 @@
 # Template Extension Specification
 
-- **Title:** Template
-- **Identifier:** <https://stac-extensions.github.io/template/v1.0.0/schema.json>
-- **Field Name Prefix:** template
+- **Title:** Classification
+- **Identifier:** <https://stac-extensions.github.io/classification/v1.0.0/schema.json>
+- **Field Name Prefix:** classification
 - **Scope:** Item, Collection
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
-- **Owner**: @your-gh-handles @person2
+- **Owner**: @drwelby @mmohr
+
+This document explains the Classification Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
+
+Classification stores metadata that clarifies the values within a dataset. Common uses would be:
+
+- Describing classes of data, and the values belonging to the class
+- The reverse of the above, as a lookup table mapping values to classes
 
-This document explains the Template Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
-This is the place to add a short introduction.
 
 - Examples:
-  - [Item example](examples/item.json): Shows the basic usage of the extension in a STAC Item
+  - [Asset example](examples/item.json): Shows the basic usage of the extension in a STAC Item
   - [Collection example](examples/collection.json): Shows the basic usage of the extension in a STAC Collection
 - [JSON Schema](json-schema/schema.json)
 - [Changelog](./CHANGELOG.md)
 
-## Item Properties and Collection Fields
+## Asset Properties 
+
+For single-band rasters
+
+## Raster Band (Raster Extension)
+
+For multiband rasters
+
+## Table Column (Table Extension)
+
+For tabular or vector datasets
+
+## Collection Fields
+
+In `summaries` field
+
+
+_naming the field "classes" seems more descriptive but ??_
 
 | Field Name           | Type                      | Description |
 | -------------------- | ------------------------- | ----------- |
-| template:new_field   | string                    | **REQUIRED**. Describe the required field... |
-| template:xyz         | [XYZ Object](#xyz-object) | Describe the field... |
-| template:another_one | \[number]                 | Describe the field... |
+| classification:classes   | [Class object]        | **REQUIRED**. Classes in the dataset |
 
-### Additional Field Information
 
-#### template:new_field
+_Current fields from file:values_
+
+### Class Object
+
+| Field Name           | Type                      | Description |
+| -------------------- | ------------------------- | ----------- |
+| values  | [Any] | **REQUIRED** Values in the class |
```

Comment:
From a recent conversation, mentioned by @alexgleith - Here’s how the ODC defines bit flags:

- https://github.com/digitalearthafrica/config/blob/master/products/ls9_sr.odc-product.yaml#L55
- S2: https://github.com/digitalearthafrica/config/blob/master/products/esa_s2_l2a.odc-product.yaml#L84